examples/mysql-wordpress-pd/local-volumes.yaml

```diff
@@ -1,3 +1,9 @@
+# When using hostPath and selinux the mysql container will not be able to `chown /var/lib/mysql`
+# due to a permissions error. To remedy this without setting selinux to permissive:
+#   1) the hostPath PVs are located in /tmp/data rather than /tmp:
+#        on host: `mkdir -p /tmp/data & chmod a+rwt /tmp/data`,
+#   2) set the selinux label type to "svirt_sandbox_file_t" on /tmp/data:
+#        on host: `chcon -Rt svirt_sandbox_file_t /tmp/data`
```

Comment:
Remove the comment. Lets keep the configs clean and the explanation in the README.